spec/secrets.yml

```
@@ -93,7 +93,7 @@ components:
 
 
           Note: Conjur will allow you to add a secret to any resource, but the best practice is to store and retrieve secret data only using Variable resources."
-        operationId: "getVariable"
+        operationId: "show"
```

I don't know if `show` is the right verb here

we use `show` above to get the resource / role info (id / policy path / annotations / etc)

here we are doing something slightly different, we are retrieving the variable value

in the CLI we use `get` as the verb
```suggestion
        operationId: "get"
```